deps/osxunwind.def

```diff
@@ -0,0 +1,2 @@
+-nr:___register_frame:___jl_register_frame
+-nr:___deregister_frame:___jl_deregister_frame
```

Comment:
that would be fine as well. Though we do use it a lot, so it would also be somewhat inconvenient.